macros/AppletObjects.pl

```diff
@@ -71,6 +70,7 @@ =head3  GeogebraWebApplet
 =cut
 
 sub GeogebraWebApplet {
+	ADD_JS_FILE("//web.geogebra.org/4.4/web/web.nocache.js", 0);
```

Comment:
Should this be a full URL? Is it necessary to have a fixed version number in the path?

Reply:
Yes, it is a full URL, and needs to be.  Or are you referring to the lack of the http or https protocol?

It would probably be nice to have the link be pulled in from a setting somewhere, but it was done this way before.  I didn't change that.  See line 1163 in pg/lib/Applet.pm.

Reply:
Actually, I remember when I was working on this pull request that I looked at the newer Geogebra javascript API, and I recall that it will not work with the applet code in webwork2.  I think that the version is more tightly bound to the way the macro utilizes the API, and so it probably does need to be hard coded in in this way.  At least for now, until someone wants to figure out a better way to do it.

Reply:
I was referring to the missing http/https.I guess there may be reasons not to force https.

It would certainly be nice to get the URL from a setting, but it is harder to do that in PG, and with the intention to have the standalone render pick up more uses and possible to try in a future version of WW to split the renderer out of the Apache process - it is probably not a very good idea to try to pass it from webwork2 into pg.

For now - if it works - I guess we should leave it as you did it, and we can look into how to make it configurable in the future.